include/gsl/multi_span

```diff
@@ -14,10 +14,8 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#pragma once
-
-#ifndef GSL_MULTI_SPAN_H
-#define GSL_MULTI_SPAN_H
+#ifndef GUIDELINE_SUPPORT_LIBRARY_SPAN_H
```

Comment:
Shouldn't this be `GUIDELINE_SUPPORT_LIBRARY_MULTI_SPAN_H`?

Reply:
Fixed